firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,156 @@ extension String {
+extension String {
+    public struct EnhancedTrackingProtection {
+        public static let onTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%@ is on guard",
+            comment: "Title for the enhanced tracking protection screen when the user has selected to be protected. The placeholder will have the value of the app name")
+
+        public static let onHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "You’re protected. If we spot something, we’ll let you know.",
+            comment: "Header for the enhanced tracking protection screen when the user has selected to be protected.")
+
+        public static let offTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Off.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "You turned off protections",
+            comment: "Title for the enhanced tracking protection screen when the user has opted out of the feature.")
+
+        public static let offHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Off.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Firefox is off-duty. We suggest turning protections back on.",
+            comment: "Header for the enhanced tracking protection screen when the user has opted out of the feature.")
+
+        public static let onNotSecureTitle = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.NotSecure.Title.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Be careful on this site",
+            comment: "Title for the enhanced tracking protection screen when the user has selected to be protected but the connection is not secure.")
+
+        public static let onNotSecureHeader = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.On.Header.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Your connection is not secure.",
+            comment: "Header for the enhanced tracking protection screen when the user has selected to be protected but the connection is not secure.")
+
+        public static let connectionVerifiedByLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Verifier.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Verified by %@",
+            comment: "String to let users know the site verifier, where the placeholder represents the SSL certificate signer.")
+
+        public static let trackersBlockedLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%i Trackers blocked",
```

Comment:
do we want this to be %@ instead?

Reply:
this will be an integer but I made it %@ for consistency

Reply:
I think %i would be okay if you also want to make the other integer values follow suite.